WebdriverIO/tests/response-validation.spec.ts

```diff
@@ -53,6 +59,13 @@ describe('Response validation', () => {
         request = await browser.getRequest(0);
         expect(request.response.body).to.deep.contain(users.originalPartialUsersBody);
       });
+
+      it('should appear on the UI', async () => {
+        request = await browser.getRequest(0);
```

Comment:
You dont seem to use this request anywhere in this testcase, so this seems like an unnecessary step.
Also regarding this request variable, wouldnt it be better to give it a value in a before hook? 
i believe the value is the same through the whole test suite.